Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -15,8 +15,75 @@
 import SwiftUI
 import ArcGIS
 
-/// The `OfflineMapAreasView` component displays a list of downloadable
-/// preplanned map areas or on-demand map areas from a given web map.
+/// The `OfflineMapAreasView` displays a list of downloadable preplanned or
+/// on-demand map areas from a given web map.
+///
+/// The view allows users to download, view, and manage offline maps
+/// when network connectivity is poor or nonexistent.
+///
+/// **Features**
+///
+/// The view supports both ahead-of-time(preplanned) and on-demand workflows
+/// for offline mapping. For both workflows, the view:
+///
+/// - Opens a map area for viewing when selected.
+/// - Shows download progress and status for map areas.
+/// - Provides options to view details about downloaded map areas.
+/// - Supports removing downloaded offline map areas files from the device.
```

Comment:
This feels out of sequence. We should first talk about displaying or creating map areas.
Then downloading, and opening.